examples/gno.land/p/samourai/project_manager/project_manager.gno

```diff
@@ -0,0 +1,183 @@
+package project_manager
+
+import (
+	"std"
+
+	"gno.land/p/demo/avl"
+)
+
+const (
+	DivisionTech = "tech"
+	DivisionProd = "prod"
+)
```

Comment:
we should not have sam-specific stuff here I think

Reply:
yep, was just to dogfood the realm, i went really fast on this
I will rework it if we decide to spend time on it